office-add-in/src/commands/commands.html

```diff
@@ -1,7 +1,10 @@
 <!--
-  ~ SPDX-FileCopyrightText: 2025 Info
+  ~ SPDX-FileCopyrightText: 2025 INFO
```

Comment:
```suggestion
  ~ SPDX-FileCopyrightText: 2025 INFO.nl
```

See: https://infonl.atlassian.net/wiki/spaces/TECH/pages/21922614/Open+sourcing+software+at+INFO